azure-blob-storage/README.md

```diff
@@ -0,0 +1,33 @@
+# Prerequisites
+You  should have Node.js > 7.8 installed.
+
+## Installation
+```sh
+$ npm install
+```
+
+## Usage
+ ```sh
+$ node app.js [Options] -- <api_key> <api_secret> <funnel_id>
+```
+
+### Options
+- -c, --container=STRING  The name of the azure container
+- -s, --storageAccount=STRING   The name of the storage account
+- -t, --token=STRING  SAS token
```

Comment:
SAS is the good choice here, but how do we monitor it that it does not expire?

Reply:
Still to be decided, maybe another check.

Reply:
yep, that for sure need to be another check but more complex one. Not like AD secrets, here we don't have a single place we can query to find expired SAS tokens...

Reply:
I think that if the token expires the check will fail and we'll get alert. IMO it's good enough.